core/storage/user/gae_models.py

```diff
@@ -856,13 +856,20 @@ def export_data(user_id):
         if user_model is None:
             return {}
 
+        creator_usernames = []
+        for creator_id in user_model.creator_ids:
+            creator = UserSettingsModel.get(creator_id)
```

Comment:
Can you use get_by_id here?

Better still, can you use get_multi()? Better to do just one call.

Reply:
Done